src/agent/command_store/src/command_store.cpp

```diff
@@ -0,0 +1,161 @@
+#include <command_store.hpp>
+
+#include <chrono>
+#include <iostream>
+
+namespace command_store
+{
+    constexpr double MILLISECS_IN_A_SEC = 1000.0;
+
+    Status CommandStore::StatusFromInt(const int i)
+    {
+        switch (i)
+        {
+            case 0: return Status::SUCCESS; break;
+            case 1: return Status::ERROR; break;
+            case 2: return Status::INPROCESS; break;
+            default: return Status::TIMEOUT; break;
```

Comment:
I'm not sure about this default value, maybe we should define an UNKNOWN status type.

Reply:
Done